packages/react/src/tabs/tab/TabsTab.tsx

```diff
@@ -166,6 +167,9 @@ export const TabsTab = React.forwardRef(function Tab(
         onClick,
         onFocus,
         onPointerDown,
+        onKeyDownCapture() {
+          isNavigatingRef.current = true;
+        },
```

Comment:
This is needed when moving from a disabled tab to a non-disabled one with the new effect logic, since it stays stuck on the disabled item otherwise (`selected` remains on the non-disabled tab, but the highlighted index doesn't)